api/externals/repository/purchase_report_repository.go

```diff
@@ -19,9 +19,9 @@ type PurchaseReportRepository interface {
 	Create(context.Context, string, string, string, string, string, string) error
 	Update(context.Context, string, string, string, string, string, string, string) error
 	Delete(context.Context, string) error
-	AllWithOrderItem(context.Context) (*sql.Rows, error)
-	FindWithOrderItem(context.Context, string) (*sql.Row, error)
-	GetPurchaseItemByPurchaseOrderID(context.Context, string) (*sql.Rows, error)
+	AllDetails(context.Context) (*sql.Rows, error)
+	FindDetail(context.Context, string) (*sql.Row, error)
+	GetItemInfo(context.Context, string) (*sql.Rows, error)
```

Comment:
```suggestion
	AllItemInfo(context.Context, string) (*sql.Rows, error)
```
repositoryではGet使ってないので、`All`とかで統一した方が良さそうですね